lib/omnibus/software.rb

```diff
@@ -696,6 +750,11 @@ def overridden?
       @overrides.key?(:version) && (@overrides[:version] != @version)
     end
 
+    def build_dep
+      @build_dep
+    end
+    expose :build_dep
```

Comment:
@scotthain Can you link to the existing `build_dependency` method definition? I'm having a hard time finding it.